bin/jruby+truffle.bat

```diff
@@ -0,0 +1 @@
+echo Windows are not yet supported by jruby+truffle.
```

Comment:
English is weird. You really want to say "Windows is not" here because it's a singular item referring to the name of the operating system.